tests/e2e/macvtap_test.go

```diff
@@ -231,6 +238,24 @@ func filterPods(pods []v1.Pod, filterFunction func(v1.Pod) bool) []v1.Pod {
 	return filteredPods
 }
 
+func findMacvtapNamespace() (string, error) {
+	macvtapDaemonSetName := "macvtap-cni"
+	daemonSetList, err := clientset.ExtensionsV1beta1().DaemonSets(v1.NamespaceAll).List(context.TODO(), metav1.ListOptions{
+		FieldSelector: fmt.Sprintf("metadata.name=%s", macvtapDaemonSetName),
+	})
+	if err != nil {
+		return "", errors.Wrap(err, "Failed to get daemonSet list in all namespaces")
+	}
+
+	for _, daemonset := range daemonSetList.Items {
+		if daemonset.GetName() == macvtapDaemonSetName {
+			return daemonset.GetNamespace(), nil
+		}
+	}
+
+	return "", errors.New("Failed to find macvtap daemonSet")
```

Comment:
I think there's no need for any of this now; you can just

```golang
func findMacvtapNamespace() (string, error) {
	macvtapDaemonSetName := "macvtap-cni"
	daemonSetList, err := clientset.ExtensionsV1beta1().DaemonSets(v1.NamespaceAll).List(context.TODO(), metav1.ListOptions{
		FieldSelector: fmt.Sprintf("metadata.name=%s", macvtapDaemonSetName),
	})
	if err != nil {
		return "", errors.Wrap(err, "Failed to get daemonSet list in all namespaces")
	}
        if len(daemonSetList.Items) == 0 {
                return errors.New("Failed to find macvtap daemonSet")
        }
	
	return daemonSetList.Items[0].GetNamespace(), nil
}
```

Reply:
[Done](https://github.com/kubevirt/macvtap-cni/compare/819e8ae41a9d85b81b9f2b74caedb30fd34b5193..0197a5af5d4bf34c3af59ff92868a993c4928c6e)